unixHttpRoutes.go

```diff
@@ -83,6 +83,10 @@ func AddProcessRoutes(router *httprouter.Router) {
 	router.GET("/net/events", handleSubscribeNetworkEvents)
 	router.GET("/net/events/:subscription", handleGetLatestNetworkEvents)
 
+	router.GET("/dns", handleGetDNS)
```

Comment:
I forget...did we decide to move this under /net/dns?

Reply:
I think we should for consistency purposes

Reply:
done